sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -234,7 +234,7 @@ function withClientCertificateCredential() {
 
 This example demonstrates authenticating the `SecretClient` from the [@azure/keyvault-secrets][secrets_client_library] client library using the `DeviceCodeCredential`.
 
-For more information about how to configure an AAD application for device code flow please refer to [Enable applications for device code flow][device_code_flow].
+For more information about the Device Code flow, please refer to: [MSAL Authentication flows - Device code](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-authentication-flows#device-code). For more information on how to configure an AAD application to authenticate using the device code flow please refer to: [Register a public client application in Azure Active Directory](https://docs.microsoft.com/en-us/azure/healthcare-apis/fhir/register-public-azure-ad-client-app).
```

Comment:
@ramya-rao-a how does this look? 🌞

Reply:
why are we using docs from some health care stuff to show how to do device code auth...

Reply:
oh wow, I didn’t see that. Alright, let me see what else I can find